### ISSUE TITLE
Fix CSV generation

### ISSUE THREAD

**Issue by maxf** [alphagov/govuk-knowledge-graph-search#29]
1. Dates are now shown as objects
2. Withdrawn reason (and possibly other prose) isn't properly escaped

**Comment by nacnudus** [comment]
Dates can be fixed as follows.

```ts
          // Hack in case it is a date from BigQuery.  We don't load the
          // BigQuery library here in the front end, so we can't check the
          // type or class of the field.
          if (field.constructor.name == 'Object' && "value" in field) {
            field = field.value;
          }
```

Withdrawn reasons etc. can be fixed as follows.

```ts
          // Replace newline characters, which jumble the CSV
          field = field.toString().replace(/\r?\n|\r/gm, ' ');
```

But fields that contain quotes and commas will be harder.  I think we ought to use a proper CSV library.  https://www.gov.uk/government/news/take-aim-press-print

**Comment by nacnudus** [comment]
Fixed by #36
